### ISSUE TITLE
Lots of missing leading slashes?

### ISSUE THREAD

**Issue by Nielius** [rubo77/rsync-homedir-excludes#8]
Hi Ruben,

Thanks for sharing the list. It saved me quite some time today.

I noticed that some of the entries had a leading slash, while some didn't (e.g., there are entries of the form `/Downloads`, but also entries of the form `.android`. I didn't know what this slash meant, but I looked it up (see for example [this question](http://unix.stackexchange.com/a/83403) on unix.stackexchange) and if I understand it correctly, more or less every entry should start with a slash: we (presumably) do not want to skip every folder containing `.android` _somewhere_ in its path, but we only want to skip `/home/$USER/.android`, and that is what the slash is for.

**Comment by rubo77** [comment]
Yes, for .android you are right, but for others it is wise to skip those folders everywhere

**Comment by rubo77** [comment]
What about folders like 

    Something.android

Are they ignored then too?

**Comment by DJCrashdummy** [comment]
no, why should they? only with `/*.android`, `/?*.android` or `/**.android`, else you are safe.

> [...] but for others it is wise to skip those folders everywhere.

i'm not sure if that is true! since we are talking about a kind of backup, i'm just excluding concrete files/folders "i have already seen somewhere" and i'm sure to want to exclude them.
...not to exclude some future files/folders by accident! - rather a bigger backup (with some files/foldes you don't need - especially with rsync it shouldn't be a hassle), than missing some files/folders in case of an accident.
